src/generics.rst

```diff
@@ -169,6 +169,19 @@ A :t:`constant parameter` shall be used in the following contexts:
 * :dp:`fls_byqjs5fvy2bj`
   As a :t:`value` of any :t:`expression` within a :t:`function body`.
 
+:dp:`fls_reASCId0i117`
+An :dt:`inferred constant` is a :t:`constant argument` which is inferred
+from the surrounding context by using ``_`` (underscore) in its place.
```

Comment:
It might be OK, but this phrasing is a bit odd.  "An inferred constant is a constant argument which is inferred...".  We could say, "wait, what is inferred?  The constant argument?  But I thought the inferred constant itself was the constant argument."

Maybe we could say, e.g., "an inferred constant is a constant argument whose constant value is inferred...".

This also avoids using "which" (rather than "that") for a restrictive clause, which is often proscribed.